docs/collecting-data/collecting-from-own-applications/python-tracker/emitters/index.md

```diff
@@ -22,16 +22,21 @@ This is the signature of the constructor for the base Emitter class:
 def __init__(
         self,
         endpoint: str,
-        protocol: Literal["http", "https"] = "http",
+        protocol: Literal["http", "https"] = "https",
         port: Optional[int] = None,
-        method: Literal["get", "post"] = "get",
+        method: Literal["get", "post"] = "post",
         buffer_size: Optional[int] = None,
         on_success: Optional[Callable[[PayloadDictList], None]] = None,
         on_failure: Optional[Callable[[int, PayloadDictList], None]] = None,
         byte_limit: Optional[int] = None,
         request_timeout: Optional[Union[float, Tuple[float, float]]] = None) -> None:
 ```
 
+:::note Prior to v0.12.0
+
+Before version 0.12.0 the default values for the emitter `protocol` and `method` were `http` and `get` respectively.
+
+:::
 | **Argument** | **Description** | **Required?** | **Type** |
 | --- | --- | --- | --- |
 | `endpoint` | The collector URI | Yes | Dict |
```

Comment:
Nit: could we add a column with the default values similar like in the [tracker parameters](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/python-tracker/intialization/#creating-a-tracker)?

Reply:
Also can you please fix the next row with protocol – I don't think one can use capital `HTTP` and `HTTPS` as it suggests. Should be something like: `Request protocol: "http" or "https"`. Also the type should not be a `List` but `String`

Reply:
And the type for the `endpoint` should be a `String` as well.